at_secondary/at_secondary_server/lib/src/exception/global_exception_handler.dart

```diff
@@ -151,6 +152,9 @@ class GlobalExceptionHandler {
 
   void _writeToSocket(AtConnection atConnection, String prompt,
       String? errorCode, String errorDescription) {
-    atConnection.write('error:$errorCode-$errorDescription\n$prompt');
+    atConnection.write('error:${jsonEncode({
+          'errorCode': errorCode,
+          'errorDescription': errorDescription
+        })}\n$prompt');
```

Comment:
NB: We can't deploy this to production until all clients have code which can cope with either the current error response or this new error response, or we have code on the server-side which can detect what capability the client has. @VJag should we consider having the client be able to send some info telling the server what version of the at_client it is running?